### ISSUE TITLE
Select Data Type Page- 508 Issue 

### ISSUE THREAD

**Issue by JanellC** [US-EPA-CAMD/PI-1#1489]
## Required Fix 

- [x] RF-1a 
- [x] RF-3a
- [x] SEF-1

## Context 

See [EPA CAMD Accessibility Report ](https://teams.microsoft.com/l/file/31E8B6EF-1313-4495-B867-46347BB7E4A2?tenantId=88b378b3-6748-4867-acf9-76aacbeca6a7&fileType=docx&objectUrl=https%3A%2F%2Fusepa.sharepoint.com%2Fsites%2FCAMDCVPTeam%2FShared%20Documents%2FDatateers%2F5-UI%20Design%2F02%20508%20and%20Accessibility%2FEPA.CAMPD.Accessibility.Report%20.docx&baseUrl=https%3A%2F%2Fusepa.sharepoint.com%2Fsites%2FCAMDCVPTeam&serviceName=teams&threadId=19:f4349e2ba95e43f3b29e8208bc8f5d5f@thread.skype&groupId=21a5e1cd-d3f1-48a6-801f-0d34e2d63d23) for how to fix and additional details 

| Issue #  | Issue Description  |
|-|-|
| RF-1a | Heading tags have been implemented but heading levels were either skipped (for example, H2 to H4) or did not move down a level when a logical subsection was present. This issue is a violation of Section 508 and WCAG 2.0 Success Criterion 1.3.1. |

| RF-3a | ARIA is used on the page but it does not fully conform to the ARIA specifications. This issue is a violation of WCAG 2.0 Success Criterion 4.1.2 |
| SEF-1 | There is a linked image where the alt text does not match or include the visual text in the image. This issue is a violation of WCAG 2.1 Success Criterion 2.5.3. |

## Testing 
 
Tester: @aprematta

**Comment by aprematta** [comment]
All changes confirmed, closing ticket
